packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -18,7 +18,7 @@ export interface OrgUserInfo {
 
 export const WORKSPACE_CONTEXT_ORG_ID_ERROR = 'workspace_context_org_id_error';
 /**
- * Manages the context of a workspace during a session with an open SFDX project.
+ * Manages the context of a workspace during a session with an open SF project.
```

Comment:
Again, being picky about this acronym 🙂 

```suggestion
 * Manages the context of a workspace during a session with an open Salesforce project.
```